server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/materials/git_material_representer.rb

```diff
@@ -27,6 +27,7 @@ class GitMaterialRepresenter < ScmMaterialRepresenter
                    value.blank? ? self.setBranch('master') : self.setBranch(value)
                  }
         property :submodule_folder
+        property :shallow_clone
```

Comment:
The document PR: https://github.com/gocd/api.go.cd/pull/73

Thx for pointing this out.